util/regexp-assemble/README.md

```diff
@@ -6,9 +6,15 @@
     ```bash
     git submodule update --init util/regexp-assemble/lib/lib
     ```
-3. You should now be able to use the script. Try running something like the following:
+3. You need to install some dependencies. With `virtualenv`, installing them would look like this:
```

Comment:
I know we use Python3 for all Python-based script, but I think it would be nice to list it somewhere.

Reply:
I would put it in the requirements file, I think. Does that make sense? Or did you mean to add it to the documentation explicitly?

Reply:
Looks like `util/regexp-assemble/requirements.txt` used for PIP. If PIP allows it, then it's fine by me.

Reply:
I thought that was possible but it isn't. Instead, I've documented the requirement and forced the use of Python 3 in the script shebang, as suggested by @fzipi.